packages/react/src/components/Label/Label.tsx

```diff
@@ -67,7 +63,7 @@ class Label extends UIComponent<ReactProps<LabelProps>, any> {
   static className = 'ui-label'
 
   static propTypes = {
-    ...commonPropTypes.createCommon({ color: 'complex' }),
+    ...commonPropTypes.createCommon({ color: true }),
```

Comment:
The color prop now always has a type of string, so we are removing all other color types.